.github/workflows/cypress.yml

```diff
@@ -0,0 +1,133 @@
+name: Cypress
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  end-to-end:
+    runs-on: ubuntu-20.04
+
+    services:
+      mariadb:
+        image: mariadb:10.7
+        ports:
+          - 3306:3306
+        env:
+          MARIADB_DATABASE: fusionsuite
+          MARIADB_USER: fusionsuite
+          MARIADB_PASSWORD: fusionsuite
+          MARIADB_ROOT_PASSWORD: fusionsuite
+
+    steps:
+    # Install PHP and NodeJs
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '7.4'
```

Comment:
php 7.4 is very near end of life. It could be better to use 8.0 or 8.1 directly

Reply:
I usually prefer to test with the oldest supported version (which is 7.4 according to the documentation :))

Reply:
ok ;)